src/components/LoadFileMenu/style.css

```diff
@@ -7,6 +7,23 @@
     fill: var(--dark-theme-menu-text-color);
 }
 
+.menu :global(.ant-dropdown-menu-item) {
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+.menu :global(.ant-btn-ghost) {
+    border: none;
+    color: var(--dark-theme-menu-text-color);
+    padding: 0;
+    width: calc(var(--dropdown-menu-width) - 20px);
+    text-align: left;
+}
+
+.menu :global(.ant-btn-ghost:hover) {
+    color: var(--dark-theme-menu-text-hover);
+}
```

Comment:
This was moved up from src/components/LocalFileUpload/style.css. Then I deleted that CSS file because nothing was left